modules/acquia_cms_article/tests/src/ExistingSite/BreadcrumbTest.php

```diff
@@ -88,12 +88,13 @@ public function testBreadcrumb(string $node_type, string $sub_type, array $expec
    * @return array[]
    *   Sets of arguments to pass to the test method.
    */
-  public function providerNoSubType() : array {
+  public static function providerNoSubType() : array {
+    $object = new self('test');
```

Comment:
@apathak18 I think there's no need to create an object for this. As we are calling method `providerBreadcrumb` and this method is also static, we can simply call it like: `self::providerBreadcrumb()`. Please revisit and update at all places.

Reply:
This was already merged into the base branch ~~ Now my commits are clean.